syncer/status.go

```diff
@@ -54,6 +54,10 @@ func (s *Syncer) Status() interface{} {
 		st.MasterBinlogGtid = masterGTIDSet.String()
 	}
 
+	if s.streamerController != nil {
+		st.BinlogType = binlogTypeToString(s.streamerController.GetBinlogType())
```

Comment:
1. Is it concurrency-safe to access `s.streamerController`
2. Is it needed to give any special value when `s.streamerController == nil` or just let it empty?

Reply:
it should be concurrency-safe and add some read lock in 5f5613a, and set binlogtype to `unknown` if  streamerController is nil

Reply:
Is the access of `s.streamerController` itself still not concurrency-safe (not `s.streamerController.GetBinlogType`)? seems `s.streamerController` may be modified in _syncer.go_.

Reply:
fix in d9c6160, streamerController will not be set to nil in syncer

Reply:
Reading and writing `s.streamerController` (not set to `nil` but set a new value) is acceptable concurrently, right?

Reply:
yes, only set `streamerController` from nil to a new value, and this only happened at the first time.